gcs/electron/main.ts

```diff
@@ -256,34 +254,34 @@ app.on('activate', () => {
 
 app.whenReady().then(() => {
   createLoadingWindow()
+  // Open file
   ipcMain.handle('fla:open-file', openFile)
-  ipcMain.handle('fla:get-fgcs-logs', async () => {
-    const fgcsLogsPath = path.join(os.homedir(), 'FGCS', 'logs')
+  // Get Recent Logs
```

Comment:
these comments can be combined into one `// Open file and get recent logs` would look a little cleaner